nibabel/nifti1.py

```diff
@@ -1260,7 +1260,7 @@ def set_dim_info(self, freq=None, phase=None, slice=None):
         This is stored in one byte in the header
         '''
         for inp in (freq, phase, slice):
-            if inp not in (None, 0, 1, 2):
+            if inp is not None and inp not in (0, 1, 2):
```

Comment:
As the commit message says, it came up whilst I was testing the code below.

At least with python 3.4, every time this function gets used, we get python FutureWarnings about comparing None with == (which is what in does).  The only way I could see to work around it was to split None out of the in - better suggestions welcome.